data/paradise world jobs.txt

```diff
@@ -365,3 +365,23 @@ mission "Pleasure cruise security"
 		dialog "You bid goodbye to the captain of the <npc> and accept your payment of <payment>."
 		payment
 		payment 50000
+
+
+mission "Premium materials to <planet>"
+	job
+	repeat
+	deadline
+	description "Deliver <cargo> for new high-end condominium on <destination> by <date>. Payment is <payment>."
+	cargo "Premium materials" 25 50
+	to offer
+		random < 10
+	source
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes rich
+	destination
+		distance 2 5
+		attributes paradise
+	on complete
+		payment
+		payment 25000
+		dialog "Your ship is directed to the rear loading dock of a palatial complex where your cargo of premium building materials is unloaded and maticulously inspected. You collect your payment of <payment>."
```

Comment:
> dialog "Your ship is directed to the rear loading dock of a palatial complex where your cargo of premium building materials is unloaded and maticulously inspected. You collect your payment of <payment>."

would have written it as

> dialog "Your ship is directed to the rear loading dock of a palatial complex where your cargo of premium building materials is unloaded and meticulously inspected before you collect your payment of <payment>."

Reply:
In my opinion, that makes the sentences too long; we need to take a mental breath. Instead, I would keep it split as two sentences, but change the start of the second to briefly introduce whomever you're giving the supplies to:
> The condominium employees give you your payment of \<payment\>.

Reply:
Noted. Currently busy, so I can't exactly take some time to write up something.

Reply:
> In my opinion, that makes the sentences too long; we need to take a mental breath. Instead, I would keep it split as two sentences, but change the start of the second to briefly introduce whomever you're giving the supplies to:
> 
> > The condominium employees give you your payment of <payment>.

Thanks I made some changes along those lines.